include/sealib/measurement/bytecounter.h

```diff
@@ -1,5 +1,5 @@
-#ifndef SEALIB_BYTECOUNTER_H_
-#define SEALIB_BYTECOUNTER_H_
+#ifndef SEALIB_MEASUREMENT_BYTECOUNTER_H_
+#define SEALIB_MEASUREMENT_BYTECOUNTER_H_
```

Comment:
The ByteCounter does not fit into any of our 4 categories, so it can be put directly in the *include/* folder.